dataactbroker/permissions.py

```diff
@@ -39,7 +39,7 @@ def decorated_function(*args, **kwargs):
                                 break
 
                     finally:
-                        InterfaceHolder.closeOne(userDb)
+                        userDb.close()
```

Comment:
Yes, this decorator is creating and closing a connection.  It happens before the route gets started and calls RouteUtils.run_instance_function, which creates the connection that will be used during this route call.